src/tests/issues.targets

```diff
@@ -1310,6 +1310,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/Convert/ldind_conv/**">
             <Issue>needs triage</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/Convert/signed_overflow_conversions_are_not_treated_as_unsigned/**">
+            <Issue>https://github.com/dotnet/runtime/issues/51323</Issue>
```

Comment:
Right now the issue refers to a non-existnet (reverted) test. I will update it to refer to this test if/when this is merged (in the case of this not being merged, I will close it).